src/selectors/per-thread/index.js

```diff
@@ -46,8 +47,11 @@ export type ThreadSelectors = {|
  * This is the static object store that holds the selector functions.
  */
 const _threadSelectorsCache: { [number]: ThreadSelectors } = {};
-let _mergedThreadSelectorCacheKey: ?ThreadsKey;
-let _mergedThreadSelectorCache: ?ThreadSelectors;
+const _mergedThreadSelectorsMemoized = memoize(
+  (threadIndexes: Set<ThreadIndex>, threadsKey: ThreadsKey) =>
```

Comment:
I think this can use the type of `MixedTupleMap`. By default it's using the `TupleMap`.

Per: https://github.com/memoize-immutable/memoize-immutable